config.sample.yaml

```diff
@@ -19,6 +19,7 @@ feature_flags:
   # - deactivate_only # Only deactivate users, do not create or update them. Keep in mind LDAP is cached and all the changes made on LDAP will be written to the cache as if they where applied. Therefore, only the deactivation changes will be applied to Zitadel but **all the other changes will be lost**.
 
 # Configuration for the sources to sync from.
+# IMPORTANT: Only one source can be present at a time.
```

Comment:
I think the sample config should be split in three for this, don't want to document something that can never work.

Reply:
Okay, let's split the sample as part of the documentation commit on the feature branch.